src/components/forms/text-field/RuiTextField.styles.scss

```diff
@@ -0,0 +1,367 @@
+@use '@/styles/colors.scss' as c;
```

Comment:
do you still get errors if you have the style in the SFC?

Reply:
I did, but now style logic is less than 400 so not anymore

Reply:
So I guess I can move it back to the main component